quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/HelpCommandDistTest.java

```diff
@@ -172,7 +172,9 @@ private void assertSingleJvmStarted(CLIResult run) {
 
     private void assertHelp(CLIResult result) {
         try (NamedEnvironment env = KcNamerFactory.asWindowsOsSpecificTest()) {
-            Approvals.verify(result.getOutput());
+            // normalize the output to prevent changes around the feature toggles to mark the output to differ
+            String output = result.getOutput().replaceAll("((Disables|Enables) a set of one or more features. Possible values are: )[^.]*", "$1<...>");
```

Comment:
IMHO we should check there are at least some features printed. Something like this perhaps?
```
([\S-]+\[:v\d], )+[\S-]+\[:v\d]\.
```

Reply:
I tried to get the regex right, but it turned out to be quite complicated. I found that that it can word-wrap the options with the brackets just before the opening bracket. 

So my alternative would be to check that there are at least 30 characters of options. See current commit for a regex: 

```
((Disables|Enables) a set of one or more features. Possible values are: )[^.]{30,}
```

Reply:
Ok, let's not overcomplicate things. That compromise works for me. Thanks!